plib/library/IpAddress.php

```diff
@@ -34,8 +36,58 @@ public static function getAvailable()
         throw new pm_Exception("Unable to get server IP: empty result.");
     }
 
+    public static function getAvailablePublic()
+    {
+        if (isset(self::$_publicIpAddresses)) {
+            return self::$_publicIpAddresses;
+        }
+        self::$_publicIpAddresses = [];
+
+        $request = "<ip><get/></ip>";
+        $response = pm_ApiRpc::getService('1.6.5.0')->call($request);
```

Comment:
lets avoid copy-paste of api calls. we could have only one static array [ $private => $public ] addresses